string next_state

EndCondition end_condition
